gr-azure-software-radio/python/integration_keyvault.py

```diff
@@ -0,0 +1,38 @@
+#!/usr/bin/env python
```

Comment:
New python file: Does this pass pylint?

Reply:
Passes flake8

Reply:
checking for pylint compliance

Reply:
Steve agreed to fix this with his merge.